python/pyarrow/_parquet.pyx

```diff
@@ -1665,6 +1919,82 @@ cdef shared_ptr[ArrowWriterProperties] _create_arrow_writer_properties(
 
     return arrow_properties
 
+cdef _name_to_index_map(Schema arrow_schema):
+    cdef:
+        shared_ptr[CSchema] sp_arrow_schema
+        shared_ptr[SchemaDescriptor] sp_parquet_schema
+        shared_ptr[WriterProperties] props = _create_writer_properties()
+        shared_ptr[ArrowWriterProperties] arrow_props = _create_arrow_writer_properties(
+            use_deprecated_int96_timestamps=False,
```

Comment:
Is these default schema? or these doesn't matter?

Reply:
Yeah these don't matter much. We just create them because the `ToParquetSchema` method asks for it.